client/layout/transient-notices/index.js

```diff
@@ -15,7 +15,13 @@ import './style.scss';
 
 class TransientNotices extends Component {
 	render() {
-		const { className, notices, onRemove, onRemove2 } = this.props;
+		const {
+			className,
+			notices,
+			notices2 = [],
+			onRemove,
```

Comment:
This seems real silly... we should just wrap these in a single callback and pass it down. I realise this is just the way it was done previously so no biggy if you don't fix. I also realise this will all (hopefully) go away when notes are consolidated to a single store.